.changeset/fair-bees-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': patch
+---
+
+Deprecates `ENTITY_DEFAULT_NAMESPACE` constant in favour of `DEFAULT_NAMESPACE`.
```

Comment:
Deprecated* right?

I think we're mostly sticking to American English too? :grin: